core-api/src/main/java/com/redhat/lightblue/Response.java

```diff
@@ -61,6 +75,24 @@ public Response(BaseResponse r) {
         super(r);
     }
 
+    public void ensureResponseSizeNotTooLarge(int maxResultSetSizeB, int warnResultSetSizeB, Request forRequest) {
+        this.forRequest = forRequest;
+        this.maxResultSetSizeB = maxResultSetSizeB;
+        this.warnResultSetSizeB = warnResultSetSizeB;
+    }
```

Comment:
The sentences that make up the method names are literally synonymous. At a glance, their arguments tell more about how they are different than their names. Using something like "set thresholds" is very clear that this merely configures the thresholds.

It's not a huge deal but it seemed like a simple readability improvement.